tools/locale-builder/Driver.cs

```diff
@@ -1144,6 +1144,11 @@ void Import (XmlDocument doc, CultureInfoEntry ci)
 				case "en-NZ":
 					df.AMDesignator = "a.m.";
 					break;
+				case "ko":
+				case "ko-KP":
+				case "ko-KR":
+					df.AMDesignator = "오전";
```

Comment:
same here

Reply:
I am fine with this, it reads better ;-)